packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6942,6 +6942,8 @@ export default class Meeting extends StatelessWebexPlugin {
 
     if (streams.screenShare?.audio) {
       await this.setLocalShareAudioStream(streams.screenShare.audio);
+
+      floorRequestNeeded = this.screenShareFloorState === ScreenShareFloorStatus.RELEASED;
```

Comment:
good that tests have caught that, I've also seen this issue and commented about it in https://github.com/webex/webex-js-sdk/pull/3038